main.py

```diff
@@ -1,7 +1,72 @@
-import init_django_orm  # noqa: F401
-
+from typing import List
 from django.db.models import QuerySet
+import os
+import django
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings")
+
+django.setup()
+
+from db.models import Genre, Actor  # noqa E402
+
+
+def create_genres(genre_names: List[str]) -> List[Genre]:
+    created_genres = []
+    for genre_name in genre_names:
+        genre = Genre.objects.create(name=genre_name)
+        created_genres.append(genre)
+    return created_genres
+
+
+def create_actors(actors_data: List[dict]) -> List[Actor]:
+    created_actors = []
+    for actor_data in actors_data:
+        actor = Actor.objects.create(**actor_data)
+        created_actors.append(actor)
+    return created_actors
+
+
+def main() -> QuerySet[Actor]:
+    # Create genres
+    genre_names = ["Western", "Action", "Dramma"]  # noqa
+    created_genres = create_genres(genre_names)
+
+    # Create actors
+    actors_data = [
+        {"first_name": "George", "last_name": "Klooney"},
+        {"first_name": "Kianu", "last_name": "Reaves"},
+        {"first_name": "Scarlett", "last_name": "Keegan"},
+        {"first_name": "Will", "last_name": "Smith"},
+        {"first_name": "Jaden", "last_name": "Smith"},
+        {"first_name": "Scarlett", "last_name": "Johansson"}
+    ]
+    created_actors = create_actors(actors_data)
+
+    # Update
+    for genre in created_genres:  # noqa
+        if genre.name == "Dramma":  # noqa
+            genre.name = "Drama"
+            genre.save()
+
+    for actor in created_actors:
```

Comment:
don't use loop. use filter and update -   
```
 Actor.objects.filter(
        first_name="George", last_name="Kloony"
    ).update(last_name="Clooney")
```

Reply:
same everwhere

Reply:
Sorry, but it seems to me to make no sense. If I get rid of loops, I will get Solution_1 which was rejected because I didn't use loops.

Reply:
> same everwhere
Sorry, but it seems to me to make no sense. If I get rid of loops, I will get Solution_1 which was rejected because I didn't use loops.